content/develop/interact/search-and-query/deprecated/development.md

```diff
@@ -54,47 +54,49 @@ You can replace `debian:bullseye` with your choice of OS, with the host OS being
 
 ## Installing prerequisites
 
-To build and test RediSearch you need to install several packages, depending on the underlying OS. The following OSes are supported: Ubuntu/Debian, CentOS, Fedora, and macOS.
-
-First, enter the `RediSearch` directory and then run:
+To build and test RediSearch you need to install several packages, depending on the underlying OS. The following OSes are supported:
+- Ubuntu 18.04
+- Ubuntu 20.04
+- Ubuntu 22.04
+- Debian linux 11
+- Rocky linux 8
+- Rocky linux 9
+- Amazon linux 2
+- Mariner 2.0
+- MacOS
+
+To install the prerequisites on your system using a setup script, first enter the `RediSearch` directory and then run:
 
 ```
-./sbin/setup
-bash -l
+cd ./install
+./install_script.sh sudo
+./install_boost.sh 1.83.0
 ```
 
-Note that this will install various packages on your system using the native package manager and `pip`. It will invoke `sudo` on its own, prompting for permission.
+Note that this will install various packages on your system using the native package manager (`sudo` is not required in a docker environment).
```

Comment:
```suggestion
Note that this will install various packages on your system using the native package manager (`sudo` is not required in a Docker environment).
```